src/flake8_to_ruff/mod.rs

```diff
@@ -0,0 +1,56 @@
+//! Utility to generate Ruff's pyproject.toml section from a Flake8 INI file.
+
+use std::collections::HashMap;
+
+use anyhow::Result;
+
+use crate::settings::options::Options;
+use crate::settings::pyproject::Pyproject;
+
+mod parser;
+
+pub fn convert(config: HashMap<String, HashMap<String, Option<String>>>) -> Result<Pyproject> {
+    // Extract the Flake8 section.
+    let flake8 = config
+        .get("flake8")
+        .expect("Unable to find flake8 section in INI file.");
+
+    // Parse each supported option.
+    let mut options: Options = Default::default();
+    for (key, value) in flake8 {
+        match key.as_str() {
+            "line-length" | "line_length" => match value.clone().unwrap().parse::<usize>() {
+                Ok(line_length) => options.line_length = Some(line_length),
+                Err(e) => eprintln!("Unable to parse '{key}' property: {e}"),
+            },
```

Comment:
It looks like this should be `max-line-length`/`max_line_length` instead?

https://flake8.pycqa.org/en/latest/user/options.html#cmdoption-flake8-max-line-length

Reply:
Thanks!